stable/rabbitmq/templates/statefulset.yaml

```diff
@@ -53,6 +53,20 @@ spec:
 {{ toYaml .Values.tolerations | indent 8 }}
       {{- end }}
       terminationGracePeriodSeconds: 10
+      {{- if and .Values.volumePermissions.enabled .Values.persistence.enabled .Values.securityContext.enabled }}
+      initContainers:
+      - name: volume-permissions
+        image: "{{ template "volumePermissions.image" . }}"
+        imagePullPolicy: {{ default "" .Values.volumePermissions.image.pullPolicy | quote }}
+        command: ["/bin/chown", "-R", "{{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }}", "/opt/bitnami/rabbitmq/var/lib/rabbitmq/"]
```

Comment:
I'd use sth like:

```yaml
        command: ["/bin/chown", "-R", "{{ .Values.securityContext.runAsUser }}:{{ .Values.securityContext.fsGroup }}", "{{ .Values.persistence.path }}"]
```

To do so, you need to create the parameter `persistence.path` and add it to `values.yaml` and `values-production.yaml` files (and README.md)

Reply:
This makes sense, will do